homework/02LessonTwo.js

```diff
@@ -84,6 +104,12 @@ function lessThanOrEqualTo(parameter1, parameter2) {
 // complete isEven() by returning a boolean expression
 
 function isEven(number) {
+  if(number % 2 === 0){
+    return true;
+  }
+  else{
+    return false;
+  }
 }
```

Comment:
You could also return the boolean expression that you are testing with your **if** statement:

```javascript
function isEven(number) {
  return number % 2 === 0;
}
```